harness/determined/cli/task.py

```diff
@@ -217,6 +301,105 @@ def logs(args: Namespace) -> None:
                     *common_log_options,
                 ],
             ),
+            Cmd(
+                "create",
+                create,
+                "create task (EXPERIMENTAL: This command should only be used in dev environments)",
+                [
+                    Arg("config_file", type=FileType("r"), help="task config file (.yaml)"),
+                    Arg(
+                        "--context",
+                        "-c",
+                        type=Path,
+                        help=ntsc.CONTEXT_DESC,
+                    ),
+                    Arg(
+                        "-i",
+                        "--include",
+                        action="append",
+                        default=[],
+                        type=Path,
+                        help=ntsc.INCLUDE_DESC,
+                    ),
+                    Arg("--project_id", type=int, help="place this task inside this project"),
+                    Arg("--config", action="append", default=[], help=ntsc.CONFIG_DESC),
+                    Arg(
+                        "-f",
+                        "--follow",
+                        action="store_true",
+                        help="follow the logs of the task that is created",
+                    ),
+                    Arg("--fork", type=str, help="id of parent task to fork from"),
+                    Arg(
+                        "-p",
+                        "--parent",
+                        type=str,
+                        help="task id of parent task",
+                    ),
+                    Arg(
+                        "--inherit_context",
+                        action="store_true",
+                        help="inherit the context directory of the parent task (--parent flag required)",
+                    ),
+                ],
+            ),
+            Cmd(
+                "config",
+                config,
+                "get config for given task",
```

Comment:
I am not sure what the intention is for releasing this feature, but if `det task create` is "only for devs" then probably the rest of these new endpoints should also be suppressed from help output until the whole feature is ready for public consumption?